ChangeLog.md

```diff
@@ -1,4 +1,16 @@
 ## Changelog
+
+3.3.0
+* BREAKING: shutdown() function on client now destroy the client and future requests throw error
+* Add test cases to Client
+* Improve verbose messages
+* Add request timeout handler
```

Comment:
Could mention something like `Add a new config setting requestTimeout (defaults to 5000 milliseconds)` to make it easier to know what changed when upgrading this

Reply:
I also included in other places that it should be mentioned.